homeassistant/components/spotify/media_player.py

```diff
@@ -474,6 +476,9 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
         """Implement the websocket media browsing helper."""
 
         if not self._scope_ok:
+            _LOGGER.debug(
+                "Spotify scopes are not set correctly, this can impact features such as media browsing."
+            )
```

Comment:
I'm still confused but this warning. It doesn't add much.

A) the user has to be running in debug mode
B) the user already has been served a re-auth, right?

Reply:
A) yes. Not sure if you read the linked issue, but one of the first steps we took was to enable debug mode to get more info on the issue. However there were no debug logs in place. This made it a little bit harder to pin point the exact issue. So next time there is an issue that is due too scopes, there will be at least a log entry indicating it and can save us some time.

B) In this case, the user was also re-auth but it was still broken. I'm not sure why the Spotify API was returning more scopes then needed. But it could easily be as well that 1 scope can go missing even after re-authing. Enabling debug mode will point this out if an user creates another Spotify related issue.

All in all, I want this debug entry to just make it easier to spot if there is an issue with scopes again in the future.